xbmc/utils/ActorProtocol.h

```diff
@@ -51,7 +51,9 @@ class Message
   bool Reply(int sig, void *data = NULL, int size = 0);
 
 private:
-  Message() {isSync = false; data = NULL; event = NULL; replyMessage = NULL;};
+  Message()
+  : signal(0), isSync(false), isSyncFini(false), isOut(false), isSyncTimeout(false),
```

Comment:
if you want three states then switch to an int or better yet an enum. As long as this is a bool initializing it is valid. It's not like we can see that it's not initialized anyway since it'll just randomly be true or false.